lib/IRGen/GenClass.h

```diff
@@ -168,14 +168,6 @@ namespace irgen {
                                     llvm::Value *selfValue,
                                     llvm::Value *metadataValue);
 
-  /// We emit Objective-C class stubs for non-generic classes with resilient
-  /// ancestry. This lets us attach categories to the class even though it
-  /// does not have statically-emitted metadata.
-  bool hasObjCResilientClassStub(IRGenModule &IGM, ClassDecl *D);
-
-  /// Emit a resilient class stub.
-  void emitObjCResilientClassStub(IRGenModule &IGM, ClassDecl *D);
-
```

Comment:
I'm not sure why these needed to get promoted to IGM.  I'd been intentionally not putting this kind of method there (1) so that IGM can be more meaningfully encapsulated and (2) to avoid IGM having literally 500 methods with no real organization to them.